src/View.Uno.Shared/Extensions/TextBox/FormattingTextBoxBehavior.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if NETFX_CORE || __ANDROID__ || __IOS__ || __WASM__ || WINDOWS_UWP
+﻿#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__ || WINDOWS || WINDOWS_WINUI || WINUI
```

Comment:
Does this lead to the file being included in any unwanted platform? Which platforms does it currently build on?

Reply:
Is there a way to check for all platforms? I mostly copied the symbols from `TextBoxBehavior` and tried to build solution.

Reply:
look at the csproj https://github.com/nventive/Nventive.View/blob/master/src/View.Uno.WinUI/View.Uno.WinUI.csproj

Reply:
I switched through every platform and there was no errors/issues, I'm thinking of removing the `#if` like JP said.

Reply:
Tested without the `#if` and it works on all the platforms.